ironfish/src/network/peers/peerManager.test.ts

```diff
@@ -558,13 +450,13 @@ describe('PeerManager', () => {
           sdpMid: '0',
         },
       })
-      expect(sendSpy).toBeCalledTimes(1)
+      expect(sendSpy).toBeCalledTimes(2)
```

Comment:
Marking this to check for later -- The peer was already in the connected state here, so what's triggering more than one send? Regardless, the test should be cleaned up a bit so it's just ensuring that the webRTC signaling data gets sent out, otherwise it'll be confusing why send is getting called twice.